src/frontend/src/components/VerticalDetailDisplay.tsx

```diff
@@ -16,13 +16,34 @@ interface VerticalDetailDisplayProps {
 const VerticalDetailDisplay: React.FC<VerticalDetailDisplayProps> = ({ label, content, boxStyle }) => {
   const theme = useTheme();
   const backgroundColor = theme.palette.mode === 'dark' ? theme.palette.grey[800] : theme.palette.grey[200];
+
+  // Box Styles
+  const customSx = {
+    overflow: 'auto',
+    whiteSpace: 'nowrap',
+    backgroundColor: backgroundColor,
+    justifyContent: 'center',
+    boxShadow: 1,
+    borderRadius: '10px'
+  };
+
+  // Custom Scrollbar Styles
+  const hoverSx = {
+    '&::-webkit-scrollbar': {
+      height: '0.6rem' // Adjust the the thickness of the scrollbar
+    },
+    '&::-webkit-scrollbar-thumb': {
+      backgroundColor: '#EF4345', //FinishLine 'red' color
+      borderRadius: '0.5rem'
+    },
+    '&::-webkit-scrollbar-thumb:hover': {
+      backgroundColor: '#b0191a' // Change to a darker shade of red on hover
+    }
+  };
+
   return (
-    <Box
-      overflow={'hidden'}
-      whiteSpace={'nowrap'}
-      sx={{ backgroundColor: backgroundColor, borderRadius: '10px', justifyContent: 'center', boxShadow: 1, ...boxStyle }}
-    >
-      <Typography textOverflow={'ellipsis'} textAlign={'center'} fontSize={50}>
+    <Box overflow={'auto'} whiteSpace={'nowrap'} sx={{ ...customSx, ...hoverSx }}>
+      <Typography textOverflow={'ellipsis'} textAlign={'center'} fontSize={'2.5rem'} margin={'0px 10px'}>
```

Comment:
Why change the font sizes?

Reply:
I was checking how it looked on different-sized screens. its now back to the original 50.